phpstan.neon.dist

```diff
@@ -0,0 +1,15 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    level: 1
+    paths:
+        - src
+        - tests
+    bootstrapFiles:
+        - tests/bootstrap.php
+    treatPhpDocTypesAsCertain: false
+    checkGenericClassInNonGenericObjectType: true
+    checkMissingIterableValueType: true
+    checkMissingVarTagTypehint: true
+    checkMissingTypehints: true
```

Comment:
I'm not sure if this makes sense without adding the rules: https://github.com/phpstan/phpstan-src/blob/master/conf/config.level6.neon#L10-L16

Reply:
I didn't know about this. Is `checkMissingTypehints` the only parameter to remove by now?

Reply:
I'm not 100% sure, but looks like:
- `checkMissingVarTagTypehint`: is used [at level 2 adding a service](https://github.com/phpstan/phpstan-src/blob/09abbe8bd4b634380db569efd28b682b94807232/conf/config.level2.neon#L60-L64)
- `checkMissingTypehints` and `checkMissingIterableValueType` work with [the rules from level 6](https://github.com/phpstan/phpstan-src/blob/master/conf/config.level6.neon#L10-L16).

So these configurations are apparently used when loading the corresponding services/rules, I would remove it for now

Reply:
Great! Seems that these parameters were also causing the described issue around `class Doctrine\\ORM\\Mapping\\ClassMetadata not found`.
Thank you!

Reply:
Really? 🤣 we can sleep at night now